src/main/java/com/kodilla/ecommercee/domain/Session.java

```diff
@@ -8,25 +8,27 @@
 @NoArgsConstructor
 @Getter
 @Setter
-@Entity(name = "SESSIONS")
+@Entity
+@Table(name = "SESSIONS")
 public class Session {
 
     @Id
     @NotNull
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    @Column(name ="SESSION_ID", unique = true)
-    private long sessionId;
+    @GeneratedValue
+    @Column(name ="ID", unique = true)
+    private long id;
 
-    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
According to this article @ManyToOne is EAGER as well and @OneToMany should be LAZY:

https://nullpointerexception.pl/trzy-rzeczy-ktore-powinienes-wiedziec-o-hibernate/

That was actually my mistake. I should put EAGER at the first place

Reply:
Ok, I've changed files as you written.